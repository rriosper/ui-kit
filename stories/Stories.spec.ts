import initStoryshots from '@storybook/addon-storyshots'
import { puppeteerTest } from '@storybook/addon-storyshots-puppeteer'

initStoryshots({ suite: 'Puppeter storyshots', test: puppeteerTest() })
