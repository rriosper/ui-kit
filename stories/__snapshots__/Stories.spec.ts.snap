// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Avatar Main 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 64px;
  height: 64px;
  max-width: 100%;
  max-height: 100%;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  border-radius: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: white;
  outline: none;
  border: none;
}

.c1 > img {
  width: 100%;
  height: 100%;
  border-radius: 100%;
}

.c0 {
  padding: 10px;
}

@media (prefers-color-scheme:dark) {

}

<div
  className="c0"
>
  <button
    className="c1"
  >
    OO
  </button>
</div>
`;

exports[`Storyshots Components/Button Colors 1`] = `
.c1 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c1 svg {
  width: 16px;
  height: 16px;
}

.c1:active {
  background-color: primary.D1;
}

.c1:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c2 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-ok-root,#2ECC71);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c2 svg {
  width: 16px;
  height: 16px;
}

.c2:active {
  background-color: primary.D1;
}

.c2:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c3 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-ko-root,#E74C3C);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c3 svg {
  width: 16px;
  height: 16px;
}

.c3:active {
  background-color: primary.D1;
}

.c3:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c4 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-warning-root,#E67E22);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c4 svg {
  width: 16px;
  height: 16px;
}

.c4:active {
  background-color: primary.D1;
}

.c4:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c5 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: #9b59b6;
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c5 svg {
  width: 16px;
  height: 16px;
}

.c5:active {
  background-color: primary.D1;
}

.c5:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c0 {
  padding: 10px;
}

@media (prefers-color-scheme:dark) {

}

<div
  className="c0"
>
  <button
    className="c1"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Default button
  </button>
  <br />
  <button
    className="c1"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Primary button
  </button>
  <br />
  <button
    className="c2"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    OK button
  </button>
  <br />
  <button
    className="c3"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    KO button
  </button>
  <br />
  <button
    className="c4"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Warning button
  </button>
  <br />
  <button
    className="c5"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    #9b59b6 color button
  </button>
</div>
`;

exports[`Storyshots Components/Button Disabled 1`] = `
.c1 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  opacity: 0.5;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c1 svg {
  width: 16px;
  height: 16px;
}

.c1:active {
  background-color: primary.D1;
}

.c1:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c0 {
  padding: 10px;
}

@media (prefers-color-scheme:dark) {

}

<div
  className="c0"
>
  <button
    aria-disabled={true}
    className="c1"
    disabled={true}
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
    type="button"
  >
    Disabled
  </button>
</div>
`;

exports[`Storyshots Components/Button Main 1`] = `
.c1 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c1 svg {
  width: 16px;
  height: 16px;
}

.c1:active {
  background-color: primary.D1;
}

.c1:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c0 {
  padding: 10px;
}

@media (prefers-color-scheme:dark) {

}

<div
  className="c0"
>
  <button
    className="c1"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Button Text
  </button>
</div>
`;

exports[`Storyshots Components/Button Sizes 1`] = `
.c1 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 32px;
  min-width: 32px;
  font-size: 15px;
}

.c1 svg {
  width: 16px;
  height: 16px;
}

.c1:active {
  background-color: primary.D1;
}

.c1:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c2 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 24px;
  min-width: 24px;
  font-size: 13px;
}

.c2 svg {
  width: calc(8px + 4px);
  height: calc(8px + 4px);
}

.c2:active {
  background-color: primary.D1;
}

.c2:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c3 {
  padding: 8px;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  opacity: 0.8;
  height: -webkit-fit-content;
  height: -moz-fit-content;
  height: fit-content;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
  background-color: var(--xstyled-colors-primary-root,#3498DB);
  outline: none;
  border: none;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  cursor: pointer;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  color: white;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  border-radius: 6px;
  min-height: 64px;
  min-width: 64px;
  font-size: 24px;
}

.c3 svg {
  width: 32px;
  height: 32px;
}

.c3:active {
  background-color: primary.D1;
}

.c3:hover {
  box-shadow: 0px 0px 6px #b1b1b1;
  opacity: 1;
}

.c0 {
  padding: 10px;
}

@media (prefers-color-scheme:dark) {

}

<div
  className="c0"
>
  <button
    className="c1"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Default button
  </button>
  <br />
  <button
    className="c2"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Small button
  </button>
  <br />
  <button
    className="c1"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Medium button
  </button>
  <br />
  <button
    className="c3"
    onClick={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    type="button"
  >
    Big button
  </button>
</div>
`;
